Hai Reveal: pirate weapon plot holes, A09 text walls, Teeneep continuity, and more player engagement (#7658)
endless-sky/endless-sky: data/hai/hai reveal 3 early.txt

		description `This station is made of eight Geocoris ships dismantled and reassembled into a spherical structure with external storage and landing platforms. Hai workers are shuffling about the station, installing human amenities and vast holographic projection systems. Presently it looks much like a construction site, but it is rapidly approaching completion.`

> 

Reviewer: Why?

		spaceport `The spaceport is a spacious docking assembly sheltered under broad, carapace-like structures which are arrayed around the lower decks like petals. It is designed to look impressive and still has that shiny look and freshly oiled smell of a newly minted construction, though it has only basic facilities and services.`
		security 0.7
		"required reputation" -10

# This begins with a P instead of an H so it will happen after the other Hai Reveal mission's dialog.
mission "Pirates with Hai Weapons 1: Katya's Message"
	landing
	name "Pirates with Hai Weapons"
	description "Katya wants you to find out if the Hai government is arming pirates, and if they are, to stop them. Don't tell the Hai that the Free Worlds sent you, and make sure the Republic doesn't find out."
	source Hai-home
	destination Hai-home
	to offer
		has "Hai Building Mountaintop [1b] Return: done"
	to complete
		has "Hai Reveal [A05] Take Teeneep to Hai-home: done"
	on offer
		conversation

> 			`Before you leave, Alondo finds you in a hallway. "Captain <last>, come with me. We need to have a quick conference." You follow him to an unusual room with strange padding on all sides. He types for a few seconds on a wall-mounted monitor. Soon after, you see the familiar face of Katya.`

Reviewer: ```suggestion
			`Before you leave, Alondo finds you in a hallway. "Captain <last>, come with me. We need to have a quick conference." You follow him to an unusual room with strange padding on all sides. He plugs a device with an attached keyboard into a wall-mounted monitor and types for a few seconds. Soon after, you see the familiar face of Katya.`
```

			`	"The transmission is secure," Alondo says. "This room has good soundproofing and some signal dampening. You can talk freely."`
			choice

> 				`	"What is it Katya?"`

Reviewer 2: ```suggestion
				`	"What is it, Katya?"`
```

					goto whaizzit
				`	"You've taken a lot of precautions for this call."`
			`	"Yes," she says. "This is a sensitive matter.`
			label whaizzit
			`	"There's a problem I need you to solve... discretely. The Free Worlds have seen pirates with Hai weapons for quite a while. Most weapons were on ships, but recently we've found a large stock in a warehouse on an anarchist world.`
				to display
					not "FW Bounty 3: done"
			`	"There's a problem I need you to solve... discretely. The Free Worlds have seen pirates with Hai weapons for quite a while. I know you fought the pirate ship Silverhawk back before the war. We know now that it had Hai outfits. Recently we've found a large stock in a warehouse on an anarchist world.`
				to display
					has "FW Bounty 3: done"

> 			`	"Such a vast supply could not be from individual smugglers, so this is definitely a large-scale operation. I'm worried the Hai government may be arming pirates. If they are, I hope the weapons are just misguided tribute payments. In the south, we well know how useful bounties and tributes are when dealing with pirates, but from my time as a senator, I can tell you the Republic will not be so understanding.`

Reviewer 2: ```suggestion
			`	"Such a vast supply could not come from individual smugglers, so this is definitely a large-scale operation. I'm worried the Hai government may be arming pirates. If they are, I hope the weapons are just misguided tribute payments. In the south, we well know how useful bounties and tributes are when dealing with pirates, but from my time as a councilor, I can tell you the Republic will not be so understanding.`
```

> 			`	"Such a vast supply could not come from individual smugglers, so this is definitely a large-scale operation. I'm worried the Hai government may be arming pirates. If they are, I hope the weapons are just misguided tribute payments. In the south, we well know how useful bounties and tributes are when dealing with pirates, but from my time as a councilor, I can tell you the Republic will not be so understanding.`

Reviewer: ```suggestion
			`	"Such a vast supply is unlikely to have come from individual smugglers, so we must assume it's the result of a large-scale operation. I'm worried the Hai government may be arming pirates. If they are, I hope the weapons are just misguided tribute payments. In the south, we well know how useful bounties and tributes are when dealing with pirates, but from my time as a councilor, I can tell you the Republic will not be so understanding.`
```

			`	"Yes," she says. "This is a sensitive matter.`
			label whaizzit
			`	"There's a problem I need you to solve... discretely. The Free Worlds have seen pirates with Hai weapons for quite a while. Most weapons were on ships, but recently we've found a large stock in a warehouse on an anarchist world.`
				to display
					not "FW Bounty 3: done"
			`	"There's a problem I need you to solve... discretely. The Free Worlds have seen pirates with Hai weapons for quite a while. I know you fought the pirate ship Silverhawk back before the war. We know now that it had Hai outfits. Recently we've found a large stock in a warehouse on an anarchist world.`
				to display
					has "FW Bounty 3: done"
			`	"Such a vast supply is unlikely to have come from individual smugglers, so we must assume it's the result of a large-scale operation. I'm worried the Hai government may be arming pirates. If they are, I hope the weapons are just misguided tribute payments. In the south, we well know how useful bounties and tributes are when dealing with pirates, but from my time as a councilor, I can tell you the Republic will not be so understanding.`

> 			`	"You must find out if the Hai government is arming the pirates and, if they are, put a stop to it. Don't tell them the Free Worlds asked you to do it and, whatever you do, don't let the Republic know. It could lead to an explosive diplomatic incident."`

Reviewer 2: ```suggestion
			`	"You must find out if the Hai government is arming the pirates and, if they are, put a stop to it. Don't tell them the Free Worlds asked you to do it, and whatever you do, don't let the Republic know. It could lead to an explosive diplomatic incident."`
```
It could be said both ways, but my suggestion should read better and still sound right.

					goto acceptable
					to display
						not "Pirates with Hai Weapons 1: Katya's Message: explained"
				`	"Don't worry. I'll stop the Hai from arming pirates."`
					goto acceptable
					to display
						not "Pirates with Hai Weapons 1: Katya's Message: explained"
				`	"I'll pursue that lead and see where it takes me."`
					goto acceptable
					to display
						has "Pirates with Hai Weapons 1: Katya's Message: explained"
				`	"If you vouch for her, then I'll trust her too."`
					goto acceptable
					to display
						has "Pirates with Hai Weapons 1: Katya's Message: Katya vouched for Sayari"
			label discrete
			action
				set "Pirates with Hai Weapons 1: Katya's Message: explained"
			`	You explain about the note from a diplomat who also asked for "discretion."`
			`	"A lead already! That sounds promising," says Katya.`
				goto loop
			label byname
			action
				set "Pirates with Hai Weapons 1: Katya's Message: explained"
				set "Pirates with Hai Weapons 1: Katya's Message: Katya vouched for Sayari"
			`	You explain about the note from Sayari who also asked for "discretion."`
			`	Katya seems surprised. "Sayari! Can you describe her?"`

> 			`	You explain Sayari's appearance to Katya and she responds, "Sayari has a long history of keeping peace between Hai and humans, longer than I've been alive. If she knows there are weapon deals, then I expect she wants to stop them. Still, be careful."`

Reviewer 2: How does Katya know about Sayari?

Reviewer: She's been a known entity to Humanity since first contact; assuming Katya was one of the people in the know beforehand, she should at least know of Sayari.

Reviewer 2: I suppose, but if a Dirt Belt representative would know about the Hai, why don't the other hundreds or thousands of more influential representatives know? It might be too many people to keep a secret. I'll leave it for now, because Republic governance is a bigger issue.

Author: Sayari was one of the most famous, if not the most famous, Hai among human politicians until Hai Reveal. If you knew about the Hai, you knew she was the one who made first contact. It is an unforgettable moment in human history.

Also, Katya wasn't just *a* dirt belt representative. She was the *only* representative for many planets, and was also one of the founders of the Free Worlds government and deeply involved in secret things up until she was captured. If there were councilors who knew about the Hai, she certainly would have been one of them. Given that this is Parliament, which has Republic Intelligence working for them, it would require stupidity on an astounding scale, or some pretty big plot holes, for Parliament to not know about the Hai.

I'd say there's about a 99.993% chance Katya knew about Sayari.

Reviewer 2: I know all this, but I was assuming that Katya would be much more involved and influential with Dirt Belt affairs (which sounded like a lot; piracy, famine, industrialisation, literacy, poverty, etc) than potential alien contacts, while politicians (a lot of them each planet, from the sounds of it) from highly populated northern planets like Prime, Sunracer, Shroud, etc. would certainly know. One Dirt Belt representative representing about a million people may not have the same amount of influence as those that represent billions of people. But I'll let this one go.
